opendbc/car/toyota/carcontroller.py

```diff
@@ -163,6 +164,9 @@ def update(self, CC, CS, now_nanos):
     # *** gas and brake ***
     # For cars where we allow a higher max acceleration of 2.0 m/s^2, compensate for PCM request overshoot and imprecise braking
     if self.CP.flags & ToyotaFlags.RAISED_ACCEL_LIMIT and CC.longActive and not CS.out.cruiseState.standstill:
+      actuators_accel = min(actuators.accel, self.accel_prev + ACCEL_WINDUP_LIMIT / 6 / 3)
+      self.accel_prev = actuators_accel
```

Comment:
When taking off from a stop, we now rate limit the request before everything (and slow it down) so that the PCM compensation can recognize when the internal PCM request is rising faster than us, and then allows us to compensate before we even start moving